vformer/functional/norm.py

```diff
@@ -11,11 +11,17 @@ class PreNorm(nn.Module):
         Attention class
     """
 
-    def __init__(self, dim, fn):
+    def __init__(self, dim, fn, context_dim=None):
```

Comment:
Could you please add the description of `context_dim` in the doc string? 
Thanks.

Reply:
fixed with [8a2fdcb](https://github.com/SforAiDl/vformer/pull/79/commits/8a2fdcbecc7e094f5a46edf8acb8fe217ea4d9aa)